src/frontend/screens/Settings/ProjectSettings/SyncSettings.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import {ScrollView, StyleSheet} from 'react-native';
+import {useIntl, defineMessages} from 'react-intl';
+import {SelectOne} from '../../../sharedComponents/SelectOne';
+import {SYNC_BACKGROUND} from '../../../lib/styles';
+import {
+  usePersistedSettings,
+  usePersistedSettingsAction,
+} from '../../../hooks/persistedState/usePersistedSettings';
+import {SyncActionSheet} from './SyncActionSheet';
+import {useBottomSheetModal} from '../../../sharedComponents/BottomSheetModal';
+
+const m = defineMessages({
+  syncSettingsTitle: {
+    id: 'screens.SyncSettings.title',
+    defaultMessage: 'Sync Settings',
+  },
+  syncPreviews: {
+    id: 'screens.SyncSettings.syncPreviews',
+    defaultMessage: 'Sync Previews (Photos Only)',
+  },
+  syncPreviewsDescription: {
+    id: 'screens.SyncSettings.syncPreviewsDescription',
+    defaultMessage:
+      'Photos will sync at a reduced smaller size. Device will <bold>not</bold> sync audio or video.',
+  },
+  syncEverything: {
+    id: 'screens.SyncSettings.syncEverything',
+    defaultMessage: 'Sync Everything',
+  },
+  syncEverythingDescription: {
+    id: 'screens.SyncSettings.syncEverythingDescription',
+    defaultMessage:
+      'Your device will sync <bold>all</bold> content at full size, including photos, audio, and videos.\n\nNote: This will use more storage.',
+  },
+  syncPreviewsBottomSheet: {
+    id: 'screens.SyncSettings.syncPreviewsButtonBottomSheet',
+    defaultMessage: 'Sync Previews?',
+  },
+  syncEverythingBottomSheet: {
+    id: 'screens.SyncSettings.syncEverythingButtonBottomSheet',
+    defaultMessage: 'Sync Everything?',
+  },
+  syncPreviewsDescriptionBottomSheet: {
+    id: 'screens.SyncSettings.syncPreviewsDescriptionBottomSheet',
+    defaultMessage:
+      'Your device will keep all existing data but new observations will sync in a smaller, preview size.\n\nYou will no longer sync Audio or Video.',
```

Comment:
does `\n` work as expected? based on my testing, it doesn't seem to:

<img width="200" alt="image" src="https://github.com/user-attachments/assets/16e25f68-1228-40e0-b9ff-5254894fa647">

<img width="200" alt="image" src="https://github.com/user-attachments/assets/95fa5c0a-5e82-4a7e-8c08-45f68b994c1b">

seems like the proper solution is to either create a separate message for this or add another rich format tag (similar to what we do for bold)

Reply:
That is so weird! It does work on my device as you can see from the screen shots. Good thing we test on different devices!

Reply:
hm yeah that's odd. fwiw, I do remember it working so I was surprised to see it not working for me. maybe I need to try a couple things on my end, or at least test on a different device to confirm

Reply:
think i figured out why we're seeing different things - see https://github.com/digidem/comapeo-mobile/pull/648#discussion_r1745970150